cli/src/command/generate_config.rs

```diff
@@ -137,33 +126,26 @@ impl ExonumCommand for GenerateConfig {
     fn execute(self) -> Result<StandardResult, Error> {
         let common_config: CommonConfigTemplate = load_config_file(self.common_config.clone())?;
 
-        let pub_config_path = self.output_dir.join(PUB_CONFIG_FILE_NAME);
-        let private_config_path = self.output_dir.join(SEC_CONFIG_FILE_NAME);
-        let consensus_secret_key_path = self.output_dir.join(CONSENSUS_SECRET_KEY_NAME);
-        let service_secret_key_path = self.output_dir.join(SERVICE_SECRET_KEY_NAME);
+        let public_config_path = self.output_dir.join(PUB_CONFIG_FILE_NAME);
+        let secret_config_path = self.output_dir.join(SEC_CONFIG_FILE_NAME);
+        let master_key_path = self
+            .master_key_path
+            .unwrap_or_else(|| "".into())
+            .join(MASTER_KEY_FILE_NAME);
 
         let listen_address = Self::get_listen_address(self.listen_address, self.peer_address);
 
-        let consensus_public_key = {
-            let passphrase = Self::get_passphrase(
-                self.no_password,
-                self.consensus_key_pass.unwrap_or_default(),
-                SecretKeyType::Consensus,
-            )?;
-            create_secret_key_file(&consensus_secret_key_path, passphrase.as_bytes())?
-        };
-        let service_public_key = {
-            let passphrase = Self::get_passphrase(
-                self.no_password,
-                self.service_key_pass.unwrap_or_default(),
-                SecretKeyType::Service,
-            )?;
-            create_secret_key_file(&service_secret_key_path, passphrase.as_bytes())?
-        };
+        let keys = {
+            let passphrase =
+                Self::get_passphrase(self.no_password, self.master_key_pass.unwrap_or_default());
+            let master_key_path = self.output_dir.join(master_key_path.clone());
```

Comment:
What about moving this line up, to line 131? Then the private config and StandardResult will use correct path (relative to execute directory in case of relative `master_key_path`, absolute if `master_key_path` is absolute). Now you need to do `output_dir.join(private_config.master_key_path)` every time you want to get correct master key path.

Am I missing something? Some tests would be nice, just to make sure relative/absolute paths work as desired.